README.md

```diff
@@ -331,17 +331,29 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 
   > **NOTE** ⏳ This command can take few minutes or immediately depending on your internet connection or model is cached. If you have issues connecting to Hugging Face, refer to the [Hugging Face discussion forum](https://discuss.huggingface.co/) for more details.
 
+  #### Downloading another instructLab model from Hugging Face
+
+  If you wish to download another [InstructLab model](https://huggingface.co/instructlab) from Hugging Face, for example the [`instructlab/granite-7b-lab-GGUF`](https://huggingface.co/instructlab/granite-7b-lab-GGUF) model, you can use this command with additional parameters as follows:
+
+  ```shell
+  ilab download --repository instructlab/granite-7b-lab-GGUF --filename granite-7b-lab-Q4_K_M.gguf --model-dir models
+  ```
+
+  This will place the downloaded model into the project's `./models` directory from which you can reference it on other commands.  To make this newly downloaded model the default, simply rerun the [`ilab init`](#️-initialize-ilab) command and when prompted enter `models/granite-7b-lab-Q4_K_M.gguf` for the model path.
```

Comment:
```suggestion
  This will place the downloaded model into the `models` directory from which you can reference it on other commands.  To make this newly downloaded model the default, simply rerun the [`ilab init`](#️-initialize-ilab) command and when prompted enter `models/granite-7b-lab-Q4_K_M.gguf` for the model path.
```